microsoft-edge/progressive-web-apps-chromium/how-to/index.md

```diff
@@ -0,0 +1,319 @@
+---
+description: An overview of the basics of Progressive Web Apps (PWAs), and tools for building Progressive Web Apps on Windows.
+title: Get started with Progressive Web Apps
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 09/15/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: progressive web apps, PWA, Edge, Windows, PWABuilder, web manifest, service worker, push
+---
+# Get started with Progressive Web Apps
+
+<!-- todo: make sure screenshots of DevTools are up-to-date -->
+
+Progressive Web Apps (PWAs) are web apps that are [progressively enhanced][WikiProgressiveEnhancement].  The progressive enhancements include app-like features, such as installation, offline support, and push notifications.  You can also package your PWA for app stores, such as Microsoft Store, Google Play, and Mac App Store.  The Microsoft Store is the commercial app store that's built into Windows 10.  
+
+In this overview of PWA basics, you create a simple web app and extend it as a PWA.  The finished project works across modern browsers.
+
+> [!TIP]
+> You can use [PWABuilder][PwaBuilder] to create a new PWA, enhance your existing PWA, or package your PWA for app stores.  
+
+
+<!-- ====================================================================== -->
+## Prerequisites
+
+*   Use [Visual Studio Code][VisualstudioCodeMain] to edit your PWA source code.
+*   Use [Node.js][NodejsMain] as your local web server.
+
+
+<!-- ====================================================================== -->
+## Creating a basic web app
```

Comment:
To be honest, the requirement for Node/Express always felt a little annoying. It might be a good optional step, but folks could also use an existing web project they have.

Reply:
Agreed, I don't like this much either. Express is used here simply to get a local web server running. Plus I don't expect people to come here to learn about how to build a website at all. If they're interested in PWA, they most likely already have a website that they're looking into turning into an installable app.
That said, we do need to provide folks with something they can easily download if they don't have anything handy, or just want to try adding a manifest and SW to a simple website.

One option comes to mind: using VS Code's live preview extension: https://github.com/microsoft/vscode-livepreview
This would require folks to have VS Code, but that seems like a better fit.

We could always just say "you need a web server to host your web site, and here are a few options: vs code live preview, npm http-server CLI, python SimpleHttpServer, ...)

@aarongustafson what do you think?

Reply:
Love it!